src/AirshipBindings.Portable/AirshipBindings.Portable.Abstractions/Channel/TagEditor.cs

```diff
@@ -6,7 +6,7 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace UrbanAirship.NETStandard.Push
+namespace UrbanAirship.Portable.Channel
```

Comment:
For some reason, git thinks I moved the NETStandard files into Portable. I don't know why. I don't think it is worth fixing.